akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -415,10 +415,28 @@ A snapshot of the full state, `akka.cluster.ClusterEvent.CurrentClusterState`, i
 as the first message, followed by events for incremental updates.
 
 Note that you may receive an empty `CurrentClusterState`, containing no members,
+followed by `MemberUp` events from other nodes which already joined,
 if you start the subscription before the initial join procedure has completed.
+This may for example happen when you start the subscription immediately after `cluster.join()` like below.
 This is expected behavior. When the node has been accepted in the cluster you will
 receive `MemberUp` for that node, and other nodes.
 
+Scala
+:  @@snip [SimpleClusterListener2.scala]($code$/scala/docs/cluster/SimpleClusterListener2.scala) { #join #subscribe }
+
+Java
+:  @@snip [SimpleClusterListener2.java]($code$/java/jdocs/cluster/SimpleClusterListener2.java) { #join #subscribe }
+
+To avoid receiving an empty `CurrentClusterState` at the beginning, you can use like below,
```

Comment:
you can use like below, => you can use it like below,

I think there's a missing `it`